plugins/nodemon/src/tasks/nodemon.ts

```diff
@@ -0,0 +1,72 @@
+import { ToolKitError } from '@dotcom-tool-kit/error'
+import { hookFork, styles } from '@dotcom-tool-kit/logger'
+import { Task } from '@dotcom-tool-kit/types'
+import { NodemonOptions, NodemonSchema } from '@dotcom-tool-kit/types/lib/schema/nodemon'
+import { VaultEnvVars } from '@dotcom-tool-kit/vault'
+import getPort from 'get-port'
+import nodemon from 'nodemon'
+import { Readable } from 'stream'
+
+export default class Nodemon extends Task<typeof NodemonSchema> {
+  static description = ''
+
+  static defaultOptions: NodemonOptions = {
+    entry: './server/app.js'
+  }
+
+  async run(): Promise<void> {
+    const { entry } = this.options
+    const vault = new VaultEnvVars(this.logger, {
+      environment: 'development'
+    })
+
+    const vaultEnv = await vault.get()
+    const port =
+      Number(process.env.PORT) ||
+      (await getPort({
+        port: [3001, 3002, 3003]
+      }))
+
+    if (!entry) {
+      const error = new ToolKitError(
+        `the ${styles.task('Node')} task requires an ${styles.option('entry')} option`
```

Comment:
**suggestion:** I suspect this may be a copy/paste error. Should this be `Nodemon`?

```suggestion
        `the ${styles.task('Nodemon')} task requires an ${styles.option('entry')} option`
```

I don't know how common logging the name is in other plugins, but potentially using `this.contructor.name` would avoid copy/paste in future and also allow extension without having to override methods. _However_ maybe extending tasks is frowned upon in which case ignore!

Reply:
Nice catch! I hope that we can make this error handling generic at some point; many tasks have to handle mandatory config options. I'm going to stick with hardcoding the name for now so that the message is a bit more readable and we don't want to actually have to be copy + pasting this code everywhere anyway!